shared/common-adapters/dumb.desktop.js

```diff
@@ -40,6 +41,20 @@ const checkboxMap: DumbComponentMap<Checkbox> = {
   },
 }
 
+const iconMap: DumbComponentMap<Icon> = {
```

Comment:
This will probably be pretty slow -- to avoid timing issues we're currently waiting 250ms per dumb map item. Can experiment with reducing the sleep. On the whole though, I don't see why this is necessary -- shouldn't the icons render the same each time? Won't this give the same exact results as the git file diff of the icon files?